Project/Assets/Prairie/Framework/Script/Interaction/ComponentToggleInteraction.cs

```diff
@@ -5,8 +5,9 @@
 public class ComponentToggleInteraction : PromptInteraction 
 {
 	public Behaviour[] target = new Behaviour[0];
+    public Behaviour[] editorTargetList = new Behaviour[0]; //used for the editor
```

Comment:
It seems to me this is being stored in two separate locations, both on the component, and on the editor for the component. The redundant data is a red-flag to me, and it appears this data is only repeated on this specific component.

Couldn't we just store the data entirely on the editor, and make sure we serialize it?